src/Nethermind/Nethermind.Runner/configs/zora-sepolia.json

```diff
@@ -23,7 +23,7 @@
     "EnginePort": 8551
   },
   "Pruning": {
-    "PruningBoundary": 256
+    "PruningBoundary": 192
```

Comment:
@emlautarom1 what should we settle with?

Reply:
Let's go with the change (192). If we see any issues we can roll back.